src/dtgtk/paint.h

```diff
@@ -179,6 +179,8 @@ void dtgtk_cairo_paint_zoom(cairo_t *cr, gint x, gint y, gint w, gint h, gint fl
 void dtgtk_cairo_paint_multiinstance(cairo_t *cr, gint x, gint y, gint w, gint h, gint flags, void *data);
 /** paint a simple grid icon */
 void dtgtk_cairo_paint_grid(cairo_t *cr, gint x, gint y, gint w, gint h, gint flags, void *data);
+/** paint focus peaking icon **/
```

Comment:
in order for `show_icons_in_paint_c.pl` tool to work, the comment needs to be in form of:
```c
/** TEXT TEXT TEXT */
```
or should I fix the pattern matching in `show_icons_in_paint_c.pl` :wink: